requirements.txt

```diff
@@ -1,23 +1,29 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
I have python3.7 locally, should I run it and add a commit on top?

Reply:
Fine. Or perhaps easier to do it in one of the other commits afterwards. Let me first juggle the various pull requests :-)